drivers/flash/Kconfig.nrf

```diff
@@ -27,6 +27,28 @@ config SOC_FLASH_NRF_RADIO_SYNC
 	help
 	  Enable synchronization between flash memory driver and radio.
 
+config SOC_FLASH_NRF_PARTIAL_ERASE
+	bool "Nordic nRFx flash driver partial erase"
+	default n
+	help
+	  Enable partial erae feature. Partial erase is performed in time
+	  slices instead of blocking MCU, for the time it is needed to
+	  complete operation over given area.
+	  This allows interrupting flash erase beetween operations
+	  to perform other task by MCU.
+	  This feature may also be used for better syncing flash erase
+	  operations, when compiled with SOC_FLASH_NRF_RADIO_SYNC,
+	  with Bluetooth.
+
+config SOC_FLASH_NRF_PARTIAL_ERASE_MS
+	int "Partial erase timeout in MS"
+	default 3
+	depends on SOC_FLASH_NRF_PARTIAL_ERASE
+	help
+	  Minimal timeout is 2ms, maximum should not exceed 7ms, 7.5ms
```

Comment:
OK, need to look into to word it properly.

Reply:
I mean that range above shouldn't be limited to 7 but to maybe a half of total page erase time. Default is ok.
@carlescufi Can you take voice here?

Reply:
Ah. OK
My reason for giving it range from 3-7ms is that it will return to CPU from NVM controller, and some other task may pick up. I have left the original loop that tried do another erase if there time left within the 7.5ms slot. 
When NVM returns and CPU does not pick other task, and the time selected is small enough to fit another erase cycle in what has left of 7.5ms slot, then it will attempt next erase.

I may have misunderstood how things operate here.

Reply:
I am not sure about this myself. Will defer to @cvinayak on this.